spec/units/derivatives_spec.rb

```diff
@@ -40,6 +40,12 @@ class CustomFile < ActiveFedora::Base
       end
     end
     describe "#{method}=" do
+      around do |obj|
```

Comment:
Isn't this save/restore usually done as follows?

```
around(:each) do |example|
  ..
  example.run
  ..
end
```

Reply:
`example.run` is definitely better. Rubocop complains about `(:each)`.